src/ota/ota_handler.rs

```diff
@@ -68,7 +76,7 @@ impl OTAStatus {
 }
 
 pub struct OTAHandler<'a> {
-    rauc: RaucProxy<'a>,
+    ota: Box<dyn OTA + 'a>,
```

Comment:
Could  `impl OTA` be used here? At the moment, it seems only `OTARauc` is being used (cf. https://users.rust-lang.org/t/difference-between-returning-dyn-box-trait-and-impl-trait/57640)